src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -29,7 +33,19 @@
    * @param symbols stock symbols to get last traded price for.
    * @return a list of the last traded price for each of the symbols passed in.
    */
-  @GetMapping("/tops/last")
+  @GetMapping("/tops/last?token=${spring.application.iex_token}")
```

Comment:
Do these still work? I think they use a different token and different URL than historical prices.

Reply:
![App_View_Old_Functionality_Works](https://user-images.githubusercontent.com/50341012/151399654-ecd4607f-5d46-4c25-bcd9-4531a60bd453.png)
Yeah, these do still work! In fact, they didn't work before adding this in - we needed the token to access the api (so Issue 1 passed all the tests, but threw 400 errors when trying it online)